packages/gator-permissions-snap/src/permissions/erc20TokenPeriodic/rules.ts

```diff
@@ -42,22 +43,42 @@ export const periodTypeRule: RuleDefinition<
   Erc20TokenPeriodicMetadata
 > = {
   name: PERIOD_TYPE_ELEMENT,
-  label: 'Period duration',
+  label: 'Transfer Window',
```

Comment:
No need to change this, as we are going to follow with a revamp of this dialog, but I think we are calling this "Frequency" - as per the design of the Sign Confirmation dialog https://www.figma.com/design/95At1Baf001Az3QF46ezvz/Permission-confirmations?node-id=1-4&t=yOmwK4YHiwDCFWaF-0

Reply:
updated